manifests/p/PHOENIXstudios/PC_DIMMER/5.6.0/PHOENIXstudios.PC_DIMMER.locale.de-DE.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: PHOENIXstudios.PC_DIMMER
+PackageVersion: 5.6.0
+PackageLocale: de-DE
+Publisher: PHOENIXstudios
```

```suggestion
Publisher: Dr.-Ing. Christian Nöding
```
This must match control panel